.evergreen/run-tests.sh

```diff
@@ -47,6 +47,9 @@ if [ "$SSL" != "nossl" ]; then
     export CA_PEM="$DRIVERS_TOOLS/.evergreen/x509gen/ca.pem"
 fi
 
+# For createvirtualenv.
+. .evergreen/utils.sh
```

Comment:
Can you revert the virtualenv changes? We can do https://jira.mongodb.org/browse/PYTHON-1960 and (https://jira.mongodb.org/browse/PYTHON-2405) in a new PR. That way we won't drag down this PR.